i18n/en/docusaurus-plugin-content-docs/version-5.0/02-quickStart/02quickstart.md

```diff
@@ -86,17 +86,17 @@ $ sh bin/tools.sh org.apache.rocketmq.example.quickstart.Consumer
 
 ## 5. Send and Receive Messages with SDK
 
-We can also try to use the client sdk to send and receive messages.
+We can also try to use the client sdk to send and receive messages, you can see more details from [rocketmq-clients](https://github.com/apache/rocketmq-clients).
```

Comment:
Hey @aaron-ai ,

The CI won't pass due to the Docusaurus bug of referencing links. It would automatically add the prefix of the original website to the link written in Markdown style. So I guess you could only temporarily reference a link by the HTML style at the moment.